src/role_nextroomer.js

```diff
@@ -308,11 +308,15 @@ roles.nextroomer.settle = function(creep) {
     methods.push(Creep.constructTask);
   }
 
+  // Fill energy after building is done
+  if (creep.room.controller.level >= 4) {
+    methods.push(Creep.transferEnergy);
+  }
```

Comment:
As mentioned in another comment. We should also consider the case:
RCL (5-8), destroyed structures, room needs to be revived. Are the nextroomers busy with filling the structures?
.... _thinking_ ... Should be fine.

Just want to make you aware of other situations.